modules/Tvpapi/resources/tvpapiAnalytics.js

```diff
@@ -5,14 +5,16 @@
 
 		defaultConfig: {
 			mediaHitInterval: 30,
-			startTime:null
+			startTime:null,
```

Comment:
Again - how is this relevant?

Reply:
@eitanavgil this is not my commit

Reply:
@MaksymKorniiov  - I see that now. 
@chanakabr why did you commit this into our branch?